lib/slack/events/request.rb

```diff
@@ -5,10 +5,16 @@ class MissingSigningSecret < StandardError; end
       class TimestampExpired < StandardError; end
       class InvalidSignature < StandardError; end
 
-      attr_reader :http_request
+      attr_reader :http_request,
+                  :signing_secret,
+                  :signature_expires_in
 
-      def initialize(http_request)
+      def initialize(http_request,
```

Comment:
Can we just use `options = {}` for consistency? I forget in which Ruby version the named options start getting supported, but I'd like it to be the same across the entire library (another PR can change it everywhere else).